### PR TITLE
i18n(fr): Update `guides/integrations-guide/*`from #9240

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Update `guides/integrations-guide/*` from #9240

> [!NOTE]
> ~~> [Check Links: src/content/docs/fr/guides/integrations-guide/vercel.mdx#L237](https://github.com/withastro/docs/pull/10328/files#annotation_29579465387)
Broken fragment link in src/content/docs/fr/guides/integrations-guide/vercel.mdx, line 237: The linked page does not contain a fragment with the name "#request"~~ 
> 
> Need to edit the `reference/api-reference/#request` and `guides/content-collections/#defining-the-collection-loader` (wrong link in `mdx.mdx` L101.)

> [!IMPORTANT]
> 🚨🚨🚨
> Note for myself, `mdx.mdx` line 101 : When `content-collections` will be updated by https://github.com/withastro/docs/pull/10145, https://github.com/withastro/docs/pull/10328/files#r1880456439, put this text with good link.
> 🚨🚨🚨

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

**Comment by thomasbnt** [comment]
> [!NOTE]
> > [Check Links: src/content/docs/fr/guides/integrations-guide/vercel.mdx#L237](https://github.com/withastro/docs/pull/10328/files#annotation_29579465387)
Broken fragment link in src/content/docs/fr/guides/integrations-guide/vercel.mdx, line 237: The linked page does not contain a fragment with the name "#request" 
> 
> Need to edit the `reference/api-reference/#request`.

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
You missed a few changes and unfortunately I can't add suggestions there... 😅 
* `netlify.mdx`: you forgot to remove a `<reference ...` l116
* `solidjs.mdx`: you forgot to remove a `côté serveur` l14
* `vercel.mdx`: you forgot to remove a `squoosh` l190 and a `<reference ...` l389

**Comment by ArmandPhilippot** [comment]
Otherwise for the broken link you can revert the change and we'll update all those links while updating `api-reference.mdx`... this is what I've done with some merged PR. Thanks to `linkcheck` we can retrieve all those links even if they are marked as up to date. 😉

File: src/content/docs/fr/guides/integrations-guide/mdx.mdx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/integrations-guide/mdx.mdx:1]: We miss some changes in this file:
* the [`MDX in Astro`](https://github.com/withastro/docs/blob/5d7321df418b5dfa0d0c51e04f84535914bd8626/src/content/docs/en/guides/integrations-guide/mdx.mdx?plain=1#L91) section
* removing a useless `await` (see #10332)

Comment (thomasbnt) [reply src/content/docs/fr/guides/integrations-guide/mdx.mdx:1]: The `MDX In Astro` [section is already written](https://github.com/thomasbnt/docs/blob/ebe4f7be40f353a58f2cea6dc75a39c6ba1c6c51/src/content/docs/fr/guides/integrations-guide/mdx.mdx?plain=1#L91), what I miss?

Comment (ArmandPhilippot) [reply src/content/docs/fr/guides/integrations-guide/mdx.mdx:1]: Oh sorry, I linked the wrong section... It was [`Using MDX wth content collections`](https://github.com/withastro/docs/blob/5d7321df418b5dfa0d0c51e04f84535914bd8626/src/content/docs/en/guides/integrations-guide/mdx.mdx?plain=1#L99) just below 😅

Comment (ArmandPhilippot) [reply src/content/docs/fr/guides/integrations-guide/mdx.mdx:1]: Just found the missing PR: #10187 😉

Comment (thomasbnt) [reply src/content/docs/fr/guides/integrations-guide/mdx.mdx:1]: Just pushed here, thanks @ArmandPhilippot. Also have a little error link from the file `fr/guides/content-collections/#defining-the-collection-loader`.

**Comment by thomasbnt** [comment]
> [!WARNING]
>  Little error link from the file` fr/guides/content-collections/#defining-the-collection-loader`. Need update for `content-collections`

**Comment by ArmandPhilippot** [comment]
Hum... we have a problem here. The PR for `content-collections` needs `integrations-guide/markdoc.mdx` to be up to date... another circular dependency! 😅 See: #10145

What we could do to make this PR mergeable: remove the link to `content-collection` in `mdx`, I already have a big alert "Notes for myself" in #10145, I can add another one to add back this link...
The PR also depends on the `tutorial` update (I'm on it), then it should be mergeable. So the link won't be away for long...

What do you think about it?

If this solution suits you, you also need to revert the link to `api-reference` in `vercel.mdx` (another broken link), we'll update it at the same time as `api-reference.mdx`.

**Comment by thomasbnt** [comment]
> Hum... we have a problem here. The PR for `content-collections` needs `integrations-guide/markdoc.mdx` to be up to date... another circular dependency! 😅 See: #10145
> 
> What we could do to make this PR mergeable: remove the link to `content-collection` in `mdx`, I already have a big alert "Notes for myself" in #10145, I can add another one to add back this link... The PR also depends on the `tutorial` update (I'm on it), then it should be mergeable. So the link won't be away for long...
> 
> What do you think about it?
> 
> If this solution suits you, you also need to revert the link to `api-reference` in `vercel.mdx` (another broken link), we'll update it at the same time as `api-reference.mdx`.



> [!IMPORTANT]
> Note for myself, `mdx.mdx` line 101 : When `content-collections` will be updated, put this text with good link.
> 🚨🚨🚨

**Review by thomasbnt — COMMENTED** [review]
See message

**Review by ArmandPhilippot — APPROVED** [review]
LGTM!

**Pull request merged**